components/kubeflow-pipeline/pipeline_components/histogram-client/flw_histogram_client/data.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+
+def data(data_id: int) -> np.ndarray:
+    np.random.seed(4242)
+    if data_id == 0:
+        local_data = np.random.normal(50.0, 15.0, size=1000)
+    if data_id == 1:
+        local_data = np.random.normal(90.0, 15.0, size=1000)
+    return local_data
```

Comment:
Let's change this to only provide a stub that's supposed to be overwritten later.

```suggestion
# This stub will be overwritten with
# client-specific implementations.
def data() -> np.ndarray:
    raise NotImplementedError

```